core/src/i18n/en.yaml

```
@@ -469,6 +474,15 @@ deployments:
   title: Deployments
 discovery-and-network:
   title: Discovery and Network
+domains:
+  exclude:
+    label: Exclude Domains
+    placeholder: Domain that should be ignored
+    tooltip: Exclude are domain names for which certificate requests are forbidden (including any subdomains) for example, *.api.mydomain.com.
+  include:
+    label: Include Domains
+    placeholder: Domain that should be observed
```

again not sure about the "should"; I assume an entry here is not just a recommendation?

```suggestion
    placeholder: Domain that is observed
```